java/ql/lib/semmle/code/java/dataflow/ListOfConstantsSanitizer.qll

```diff
@@ -0,0 +1,357 @@
+/**
```

Comment:
I think perhaps a better place for this file would be in `semmle.code.java.security`.